WcaOnRails/db/structure.sql

```diff
@@ -1086,6 +1089,7 @@ CREATE TABLE `votes` (
 /*!50001 SET character_set_results     = utf8 */;
 /*!50001 SET collation_connection      = utf8_general_ci */;
 /*!50001 CREATE ALGORITHM=UNDEFINED */
+/*!50013 DEFINER=`root`@`localhost` SQL SECURITY DEFINER */
```

Comment:
Should this go in the diff?

Reply:
Darn it! Thanks for pointing this out, this is what I get what I'm rushing...